public/locales/en/app.ftl

```diff
@@ -0,0 +1,185 @@
+-product-name = Firefox Monitor
+-brand-name = Firefox
+-brand-Quantum = Firefox Quantum
+-brand-Mozilla = Mozilla
+-brand-HIBP = HIBP
+-product-name-nowrap = <span class="nowrap">Firefox Monitor</span>
+
+layout-Firefox = {-brand-name}
+# Descriptive headline for a column of links where users can give feedback, or get additional information about, Firefox Monitor.
+layout-support = Support
+# Link that takes the user to a Firefox Monitor survey. 
+give-feedback = <span class="nowrap">Give Feedback</span>
+terms-and-privacy = <span class="nowrap">Terms and Privacy</span>
+
+error-not-subscribed = This email address is not subscribed to {-product-name}.
+error-hibp-throttled = Too many connections to {-brand-HIBP}.
+error-hibp-connect = Error connecting to {-brand-HIBP}.
+error-hibp-load-breaches = Could not load breaches.
+
+hibp-notify-email-subject = {-product-name} Alert : Your account was involved in a breach.
+
+home-title = {-product-name}
+home-not-found = Page not found.
+
+oauth-invalid-session = Invalid session
+oauth-confirmed-title = {-product-name} : Subscribed
+
+scan-title = {-product-name} : Scan Results
+
+user-add-invalid-email = Invalid Email
+user-add-email-verify-subject = Verify your subscription to {-product-name}.
+user-add-title = {-product-name} : Confirm Email
+
+user-verify-token-error = Verification token is required.
+user-verify-email-report-subject = Your {-product-name} report
+user-verify-title = {-product-name} : Subscribed
+
+user-unsubscribe-token-error = Unsubscribing requires a token.
+user-unsubscribe-token-email-error = Unsubscribing requires a token and emailHash.
+user-unsubscribe-title = {-product-name} : Unsubscribe
+user-unsubscribe-survey-title = {-product-name} : Unsubscribe Survey
+user-unsubscribed-title {-product-name} : Unsubscribed
+
+## Password Tips
+
+pwt-section-headline = Passwords = Protection
+pwt-section-subhead = Your Private Information Is Only as Safe as Your Passwords
+pwt-section-blurb =
+  Your passwords protect more than your accounts, they protect every bit of personal information that resides in them. 
+  And hackers rely on bad habits, like using the same password everywhere or using common phrases (p@ssw0rd, anyone?), 
+  so that if they hack one account, they can hack many. Here are six ways to protect your accounts.
+
+pwt-headline-1 = Use a Different Password for Every Account
```

Comment:
I think in this case the numbers are meaningful because the password tips are specifically presented in the order that they are numbered.  Additionally, the tips are displayed alongside corresponding icons which are also numbered accordingly.  I will happily defer to your judgement here if you think this is going to cause future problems though.

Reply:
Regarding the colon spacing on L13 (for some reason I can't respond to that comment directly): We just received updated copy for the emails and the extra space has been removed.

Reply:
> Additionally, the tips are displayed alongside corresponding icons which are also numbered accordingly.

How is that done? Have you considered other types of numerals?

>  I will happily defer to your judgement here if you think this is going to cause future problems though.

Two likely scenarios:
* You want to change the order. 
* You want to replace or update one of the tips. You'll need to [change the message ID](https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localization_content_best_practices#Changing_existing_strings) to do that.

In both cases you'll need to add a mapping afterward.

Also, strings are presented sequentially to localizers, so having title and description together help a bit.